statics/js/components/capture-form.js

```diff
@@ -162,12 +184,51 @@ Vue.component('capture-form', {
           self.resetQueryNodes();
           self.queryNodes = nodes;
           self.highlightQueryNodes(true);
+          if (nodes.length == 1) {
+            self.typeAllowed = true;
+            self.nodeType = nodes[0].Metadata["Type"];
+          } else if (nodes.length > 1) {
+            self.typeAllowed = false;
+          } else {
+            self.typeAllowed = true;
+          }
         })
         .fail(function() {
           self.resetQueryNodes();
+          self.typeAllowed = true;
```

Comment:
I think it should be false by default as we can't determine the node types

Reply:
done